server/events/models/models.go

```diff
@@ -385,7 +385,7 @@ type PolicySetStatus struct {
 // Summary regexes
 var (
 	reChangesOutside = regexp.MustCompile(`Note: Objects have changed outside of Terraform`)
-	rePlanChanges    = regexp.MustCompile(`Plan: (\d+) to add, (\d+) to change, (\d+) to destroy.`)
+	rePlanChanges    = regexp.MustCompile(`Plan: (?:(?:(\d+) to import, )|(?:))(\d+) to add, (\d+) to change, (\d+) to destroy.`)
```

Comment:
I think this is equivalent? (the `(?:)` matching nothing is not a pattern i've seen before 😵)

```suggestion
	rePlanChanges    = regexp.MustCompile(`Plan: (?:(\d+) to import, )?(\d+) to add, (\d+) to change, (\d+) to destroy.`)
```

Reply:
Doh yep you're right!